spec/lib/ssoe/configuration_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+# rubocop:disable RSpec/SpecFilePathFormat
+
+require 'rails_helper'
+require 'ssoe/configuration'
+
+RSpec.describe SSOe::Configuration do
+  subject(:config) { described_class.send(:new) }
+
+  let(:base_url) { 'https://int.services.eauth.va.gov:9303/psim_webservice/dev/IdMSSOeWebService' }
+  let(:cert_path) { 'spec/fixtures/certs/vetsgov-localhost.crt' }
+  let(:key_path) { 'spec/fixtures/certs/vetsgov-localhost.key' }
+
+  let(:cert_obj) { OpenSSL::X509::Certificate.new(File.read(cert_path)) }
+  let(:key_obj) { OpenSSL::PKey::RSA.new(File.read(key_path)) }
+
+  let(:faraday_connection) { instance_double(Faraday::Connection) }
+
+  before do
+    allow(IdentitySettings.ssoe_get_traits).to receive_messages(
```

Comment:
Does this not already get populated with the fields you expect? You might not even need this `before` block at all